### DISCUSSION TITLE
Error updating components?

### DISCUSSION THREAD

**Discussion by laurajimore** [uxbox/uxbox#993] [Q&A] [answered]
Hello everywhere! I am doing my first project with penpot and I only have good words. I wish the best of futures for this project.

Next I send you a capture of something that has happened to me updating a component. The modification has been made correctly to in the child-components but the image has not been shifted losing its environment. Is this a mistake or am I missing something?

I’ve updated the browser page but it not resolves, if there’s anything you can do it would be cool to know hehehe :)

VIVA PENPOT!

![image](https://user-images.githubusercontent.com/70016973/119981934-802c2400-bfbe-11eb-818b-42412ed6f8d4.png)

**Comment by elhombretecla** [comment]
Hi Laura! Thank you for your kind words and your feedback :)

We will manage this issue on Taiga. You can follow the progress here

[https://tree.taiga.io/project/penpot/issue/1744](https://tree.taiga.io/project/penpot/issue/1744)

VIVAN LOS USUARIOS DE PENPOT!

**Comment by elhombretecla** [comment]
Hi again @laurajimore

Could you check if it works now? we fixed it in the last release

**Comment by laurajimore** [comment, marked as answer]
Yes!, Good job Team :)
